neon_mq_connector/consumers/blocking_consumer.py

```diff
@@ -144,11 +146,13 @@ def join(self, timeout: Optional[float] = None) -> None:
 
     def _close_connection(self):
         try:
+            if self.is_consuming:
+                self.channel.stop_consuming()
```

Comment:
We may omit this part as in `self.connection.close()` there is internal logic that closes all of the associated consumers